metadata-ingestion/tests/unit/test_transform_dataset.py

```diff
@@ -648,18 +648,24 @@ def _test_owner(
                 )
             ]
         )
+
         transformer = ExtractOwnersFromTagsTransformer.create(
             config,
             PipelineContext(run_id="test"),
         )
-        transformed = list(
+
+        list(
             transformer.transform(
                 [
                     RecordEnvelope(dataset, metadata={}),
```

Comment:
if we put an end of stream record in here, it should do everything right?

I don't think our tests should be calling `handle_end_of_stream`